Aptabase.uplugin

```diff
@@ -1,7 +1,7 @@
 ﻿{
 	"FileVersion": 3,
-	"VersionName": "0.1.1",
-	"EngineVersion": "5.2",
+	"VersionName": "0.0.1",
```

Comment:
I would suggest to leave this as `0.1` or `0.0.1` and do tag based releases.

So when you create a `1.2.3` tag, it automatically finds and replaces it here so you don't have to manually do it every time.

Here is an example of that in action https://github.com/outoftheboxplugins/QuickActions/releases

Reply:
Sounds good, but I think the github pipeline is broken? Is there something missing on this repo?

Reply:
True, I will get on fixing it and make the appropriate releases.

I can also enable the CI as well, but that might cost quite a few compute minutes. It takes around 15 min per run.

Reply:
It's open source repo, so CI is free on github :)